### PR TITLE
feat(apikey) : Api key module in manager

### PR DESCRIPTION
**Pull request by soumyadip007**
## Does this PR introduce a breaking change
No

## Screenshot(s)

<details>
<summary>View Screenshots</summary>

image
</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/manager/components/settings/apiKey.tsx [hunk 1]
```diff
@@ -32,114 +28,81 @@ const StyledText = styled(Text)`
   --pf-c-content--h2--FontWeight: 100;
 `;
 
+const ClipboardBox = styled.div({
+  width: "500px",
+  height: "40px",
+});
```
Comment (hybridx) [inline packages/manager/components/settings/apiKey.tsx:34]:
```suggestion
const StyledClipboardBox = styled.div`
  width: 500px,
  height: 40px,
`;
```

Comment (hybridx) [reply packages/manager/components/settings/apiKey.tsx:34]: @soumyadip007 please prefix the name with Styled

File: packages/manager/pages/api/validate.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+import { getValidateUrl } from "../../utils/endpoint.utils";
+
+const Validate = async (req: AnyProps, res: AnyProps) => {
+    const url = getValidateUrl();
+    const payload = {};
+    const response = await post<AnyProps>(url, payload);
+    return res.send({ data: { token: response.token } });
+}
+
+export default Validate;
```
Comment (hybridx) [inline packages/manager/pages/api/validate.tsx:12]: new line

File: packages/manager/utils/endpoint.utils.ts [hunk 3]
```diff
@@ -18,4 +18,10 @@ export function getEventAnalyticsUrl() {
   return `${spashipHost}/event/fetch/analytics/filter`;
 }
 
+export function getValidateUrl() {
+  return `${spashipHost}/applications/validate`;
+}
 
+export function getNextValidateUrl() {
+  return `/api/validate`;
+}
```
Comment (hybridx) [inline packages/manager/utils/endpoint.utils.ts:27]: new line char

**Review by hybridx — CHANGES_REQUESTED** [review]

File: packages/manager/components/settings/apiKey.tsx (hunk 1 above)
Comment (soumyadip007) [reply packages/manager/components/settings/apiKey.tsx:34]: Done

**Review by hybridx — APPROVED** [review]

File: packages/manager/components/settings/apiKey.tsx (hunk 1 above)
Comment (hybridx) [reply packages/manager/components/settings/apiKey.tsx]: Also use template literals like i've mentioned

**Pull request merged**